### PR TITLE
Put js for table pagination back in layouts

### PR DESCRIPTION
**Pull request by gwensmu**
* Refactor gone awry
* Fix pagination bug
* Updated jump link to moods table

[#90309988]
[#90293000]

### CODE REVIEW

File: app/views/think_feel_do_engine/coach/patient_dashboards/tables/_mood_table.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     </h3>
   </div>
 
-  <div id="moodCollapse" class="panel-collapse collapse in">
+  <div id="moodsCollapse" class="panel-collapse collapse in">
```
Comment (michaelwehrley) [inline app/views/think_feel_do_engine/coach/patient_dashboards/tables/_mood_table.html.erb]: @gwensmu was it intended that this should change?  Was the corresponding href changed as well?  I was looking at https://github.com/cbitstech/think_feel_do_engine/blob/gs-fix-table-js-bug/app/views/think_feel_do_engine/coach/patient_dashboards/tables/_mood_table.html.erb#L4 and it didn't look like that was updated.  Thanks!

Comment (gwensmu) [inline app/views/think_feel_do_engine/coach/patient_dashboards/tables/_mood_table.html.erb]: @michaelwehrley fixing this right now. had made a change to the id to be consistant with the others, will fix the href

Comment (michaelwehrley) [inline app/views/think_feel_do_engine/coach/patient_dashboards/tables/_mood_table.html.erb]: @gwensmu Awesome!  After you are done, let me know, and I will merge.

**Comment by gwensmu** [comment]
@michaelwehrley just pushed fix

**Pull request merged**